rest/taskana-rest-spring/pom.xml

```diff
@@ -1,201 +1,207 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xmlns="http://maven.apache.org/POM/4.0.0"

```

Comment:
Same as the other pom: reduce the diff to the changes only.
`git checkout bb74e16187a5876ef46a91b00ae91304dda90b57 -- rest/taskana-rest-spring/pom.xml`